test/nftTrades.test.ts

```diff
@@ -61,10 +60,10 @@ describe('SwapRouter', () => {
       const foundationTrade = new FoundationTrade([foundationData])
       const methodParameters = SwapRouter.swapNFTCallParameters([foundationTrade])
       const methodParametersV2 = SwapRouter.swapCallParameters(foundationTrade)
-      registerFixture('_FOUNDATION_BUY_ITEM', methodParameters)
+      registerFixture('_FOUNDATION_BUY_ITEM', methodParametersV2)
       expect(hexToDecimalString(methodParameters.value)).to.eq(foundationData.price.toString())
       expect(methodParameters.calldata).to.eq(methodParametersV2.calldata)
-      expect(methodParameters.value).to.eq(hexToDecimalString(methodParametersV2.value))
+      expect(hexToDecimalString(methodParameters.value)).to.eq(methodParametersV2.value)
```

Comment:
did return value change for `swapNFTCallParameters` or `swapCallParameters`?

Reply:
nah its actually the call to `registerFixture` that seems to change the `value`'s type. I want to update it so it doesnt do that as that seems silly haha